### PR TITLE
Add Vimebac 0.0.0

### PR DESCRIPTION
**Pull request by smondet**

### CODE REVIEW

**Comment by smondet** [comment]
The build errors are not from this package.
Opam seems to pick the latest Tsdl (0.9.6) but the SDL library on the system is too old for it.

On my linux mint (equivalent Ubuntu LTS 16.04), I always have to `opam pin add tsdl 0.9.1` to get a Tsdl that is compatible.

**Comment by smondet** [comment]
The other kind of failure is Wall not building on Travis-OSX. Cc @let-def is this normal?

**Comment by let-def** [comment]
Ok this is a know issue with macOS <= 10.11, I will need to add another codepath for it.

**Comment by kit-ty-kate** [comment]
Sorry for not having taken care of this PR. I tried to install it manually and it gave me this error with OCaml 4.06:
```
#=== ERROR while compiling vimebac.0.0.0 ======================================#
# context      2.0.0~rc | linux/x86_64 | ocaml-base-compiler.4.06.0 | git+file:///home/opam/opam-repository
# path         ~/.opam/default/.opam-switch/build/vimebac.0.0.0
# command      ~/.opam/default/bin/jbuilder build -p vimebac -j 71
# exit-code    1
# env-file     ~/.opam/log/vimebac-889-8cb275.env
# output-file  ~/.opam/log/vimebac-889-8cb275.out
### output ###
#       ocamlc src/.vimebac.eobjs/jack_midi_source.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/default/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.vimebac.eobjs -I /home/opam/.opam/default/lib/bytes -I /home/opam/.opam/default/lib/cmdliner -I /home/opam/.opam/default/lib/ctypes -I /home/opam/.opam/default/lib/gg -I /home/opam/.opam/default/lib/grenier -I /home/opam/.opam/default/lib/integers -I /home/opam/.opam/default/lib/misuja -I /home/opam/.op[...]
# File "src/jack_midi_source.ml", line 21, characters 22-24:
# Error: This expression has type string but an expression was expected of type
#          bytes
#       ocamlc src/.vimebac.eobjs/display.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/default/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.vimebac.eobjs -I /home/opam/.opam/default/lib/bytes -I /home/opam/.opam/default/lib/cmdliner -I /home/opam/.opam/default/lib/ctypes -I /home/opam/.opam/default/lib/gg -I /home/opam/.opam/default/lib/grenier -I /home/opam/.opam/default/lib/integers -I /home/opam/.opam/default/lib/misuja -I /home/opam/.op[...]
# File "src/display.ml", line 131, characters 14-26:
# Error: This expression has type Bytes.t array
#        but an expression was expected of type string array
#        Type Bytes.t = bytes is not compatible with type string 
```

File: packages/vimebac/vimebac.0.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "nonstd"
+  "sosa"
+  "cmdliner"
+  "misuja"
+  "wall"
+  "tsdl"
+  "rresult"
+  "tgls"
+  "jbuilder" {build & >= "1.0+beta20"}
+  "ocamlfind" {build}
```
Comment (kit-ty-kate) [inline packages/vimebac/vimebac.0.0.0/opam]: Also do you need `ocamlfind` there ? I can't find any reference to either `ocamlfind` or `Findlib` in your source code

**Comment by kit-ty-kate** [comment]
Otherwise it builds fine with OCaml < 4.06

**Comment by smondet** [comment]
Thanks @kit-ty-kate , I've pushed fixes, I really need to add 4.06 to my own CI builds... :)

**Comment by kit-ty-kate** [comment]
Looks good. Thanks a lot!

**Pull request merged**